content/en/docs/how-tos/notification.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Get notifications about a job"
+date: 2020-01-06T10:08:01-04:00
+draft: false
+---
+
+The `reporter_config` stanza in a job can be used to send notifications. [For example](https://github.com/openshift/release/blob/964a2cb316ab046a8c3d856d9c99cb40bfbbef2c/ci-operator/jobs/openshift/odo/openshift-odo-master-periodics.yaml#L2-L24), the following configuration will trigger a slack notification to channel `#odo-notifications` in [coreos.slack.com](https://coreos.slack.com/) when the job is at state `success`, `failure`, or `error`. The `report_template` defines the template of the slack message.
+
+```yaml
+- agent: kubernetes
+  cluster: api.ci
+  cron: 0 */6 * * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  name: periodic-ci-openshift-odo-master-v4.2-integration-e2e-periodic
+  reporter_config:
+    slack:
+      channel: '#odo-notifications'
+      job_states_to_report:
+      - success
+      - failure
+      - error
+      report_template: '{{if eq .Status.State "success"}} :rainbow: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :rainbow: {{else}} :volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :volcano: {{end}}'
```

Comment:
We should give ppl an opportunity to figure out what variables exist and mention what happens if they use a broken template

Reply:
Yeah, ideally our example should have a much simpler, shorter `report_template`, and we should have a sentence like "you can use *these* variables" somewhere nearby, preferably with a link to upstream.

Reply:
Updated. But not sure if they look good enough.